app/models/miq_provision_virt_workflow.rb

```diff
@@ -249,6 +249,10 @@ def load_hosts_vlans(hosts, vlans)
     end
   end
 
+  def allowed_subnets(_options = {})
```

Comment:
This method will only be called by a dialog that defines the method name in `:values_from` on a dialog field and the subclassed provider workflow would include the method.

If there is common code that all, or most, workflows share it would be alright to include it here, but this not needed.

Reply:
:+1: removed